src/Symfony/Component/Intl/Resources/data/transliterator/emoji/emoji-af.php

```diff
@@ -2476,6 +2627,7 @@
     '😯' => 'gedempte gesig',
     '😲' => 'verstomde gesig',
     '😳' => 'blosende gesig',
+    '🫪' => 'verwronge gesiggie',
```

Comment:
<img width="760" height="78" alt="CleanShot 2025-11-18 at 16 52 46@2x" src="https://github.com/user-attachments/assets/a3091c28-c4a1-4784-9f27-7986ec537b79" />
his looks like this, because my browser does not support this emoji I guess, right?